RecoTracker/LSTCore/src/alpaka/LSTEvent.dev.cc

```diff
@@ -839,18 +832,16 @@ void LSTEvent::createPixelQuintuplets() {
   auto superbins_buf = cms::alpakatools::make_host_buffer<int[]>(queue_, n_max_pixel_segments_per_module);
   auto pixelTypes_buf = cms::alpakatools::make_host_buffer<PixelType[]>(queue_, n_max_pixel_segments_per_module);
 
-  alpaka::memcpy(queue_, superbins_buf, cms::alpakatools::make_device_view(queue_, pixelSeeds.superbin(), pixelSize_));
-  alpaka::memcpy(
-      queue_, pixelTypes_buf, cms::alpakatools::make_device_view(queue_, pixelSeeds.pixelType(), pixelSize_));
+  alpaka::memcpy(queue_, superbins_buf, cms::alpakatools::make_device_view(queue_, pixelSeeds.superbin()));
+  alpaka::memcpy(queue_, pixelTypes_buf, cms::alpakatools::make_device_view(queue_, pixelSeeds.pixelType()));
   auto const* superbins = superbins_buf.data();
   auto const* pixelTypes = pixelTypes_buf.data();
 
   unsigned int nInnerSegments;
   auto nInnerSegments_src_view = cms::alpakatools::make_host_view(nInnerSegments);
 
   // Create a sub-view for the device buffer
-  unsigned int totalModules = nLowerModules_ + 1;
-  auto dev_view_nSegments_buf = cms::alpakatools::make_device_view(queue_, segmentsOccupancy.nSegments(), totalModules);
+  auto dev_view_nSegments_buf = cms::alpakatools::make_device_view(queue_, segmentsOccupancy.nSegments());
```

Comment:
is it guaranteed that `totalModules` matches the column capacity (`metadata().size()`) ?